---
title: "Experiment 4 Aims and Overview"
---

## Aims 

* To understand how mutagenesis can be used in the laboratory to understand the phenotype of a microorganism

## Learning Outcomes
At the end of this laboratory session, you should understand:

*	How transposon mutagenesis can affect the phenotype of an organism.

*	How to identify the insertion site of a transposon.

*	How to predict the effects of a transposon insertion.

*	How to correlate physical mapping data (interrupted mating) and sequencing data (plasmid rescue)



## Competencies 

{{< include ./_includes/lab4comp.qmd >}}

## Importance

This practical session aligns with lectures in BM330 Fundamental Microbiology and reinforces material relating to: horizontal gene transfer, bacterial genetics. 

We have isolated a species of *Streptomyces* that produces a novel antibiotic that can kill *Escherichia coli* and other Gram-negative bacteria. We have also isolated *E. coli* strains that are resistant to this particular antibiotic and, in the previous labs, you identified the location of these antibiotic resistance mutations on the chromosome using interrupted mating.

In order to identify the mutation(s) that give rise to the antibiotic resistance phenotype, you will use a plasmid rescue experiment to identify the insertion site of the Tn*5* transposon in your mutant strain. You will map the plasmid by using restriction digest (Protocol 4.1 and Task 4A). 

To “rescue” the Tn*5* from the chromosomes of your mutant strains, we digested chromosomal DNA with the restriction enzyme AflIII, then ligated this into a similarly-digested vector plasmid. We will now check the resulting plasmid by restriction digest: plasmids with insert should produce a band upon AflIII digestion; and we also know that the enzyme HindIII cuts within the Tn*5* sequence.

Once we have verified that the isolated plasmids contain the "rescued" Tn*5*s, you will then analyse DNA sequences derived from the plasmid (Task 4B) to identify the insertion site and generate a hypothesis as to why your mutant strain is resistant to the antibiotic (Task 4C).

## Overview of Experiment 4

You will perform Protocol 4.1 and 4.2 in the Week 8 lab and your data will be posted to MyPlace for you. Perform the data analysis tasks (Tasks 4A-C) using your agarose gel data/sequence data posted on MyPlace.




